_shows/17_18/one_man_two_guvnors.md

```diff
@@ -0,0 +1,122 @@
+---
+title: One Man, Two Guvnors
+playwright: Richard Bean
+period: Autumn
+season: In House
+season_sort: 150
+date_start: 2017-12-12
+date_end: 2017-12-15
+venue: New Theatre
+
+cast:
+- role: Francis Henshall
+  name: James Curling
+- role: Alan Dangle
+  name: Chris Sharp-Paul
+- role: Stanley Stubbers
+  name: Ted Marriott
+- role: Harry Dangle
+  name: Andrew Houghton
+- role: Charlie Clench
+  name: Zack Collins
+- role: Lloyd Boateng
+  name: Dom Bitten
+- role: Alfie
+  name: Sam Morris
+- role: Gareth
+  name: Rohan Rakhit
+- role: Pauline Clench
+  name: Michaela Green
+- role: Rachel Crabbe
+  name: Gigi George
+- role: Dolly
+  name: Rachael Baines
+  
+crew:
+- role: Co-Director
+  name: George Waring
+- role: Co-Director
+  name: Kate O'Gorman
+- role: Producer
+  name: Beth Mullen
+- role: Technical Director
+  name: Jess Donn
+- role: Lighting Designer
+  name: Ben Woodford
+- role: Sound Designer
+  name: Felicity Chilver
+- role: Sound Engineer
+  name: Sam Osborne
+- role: Production Assistant
+  name: Aitor Tejo
+- role: Set Assistant
+  name: Gemma Walton
+- role: Shadow Technical Director
+  name: Shahbhan Ramzan
+- role: Shadow Lighting Designer
+  name: Miguel Barrulas
+- role: Shadow Sound
+  name: Jakub Kolek
+- role: Improv Consultant
+  name: Jack Ellis
+- role: Technical Operator
+  name: Nathan Penney
+- role: Technical Operator
+  name: Darcey Graham
+- role: Technical Operator
```

Comment:
Replace Darcey with Zoe Smith; Darcey is now a stage manager.

Reply:
Done